helm/archive-node/templates/_healthchecks.tpl

```diff
@@ -28,7 +28,9 @@ readinessProbe:
     command: [
       "/bin/bash",
       "-c",
-      "source /healthcheck/utilities.sh && isDaemonSynced && isArchiveSynced --db-host {{ template "archive-node.fullname" . }}-postgresql"
+      "source /healthcheck/utilities.sh && isArchiveSynced \
```

Comment:
Only execute this readiness check if local daemon is enabled - TODO: check out replacing with archive-node auditor tool